src/plugins/join/plugin.ts

```diff
@@ -126,7 +124,7 @@ export class Join extends ProposalPlugin<
 
     const baseState = Plugin.itemMapToBaseState(context, item)
 
-    const fundingRequestParams = {
+    const joinParams = {
```

Comment:
why fundingRequestParams replaced to joinParams ?

Reply:
It's join plugin file, it's suppose to be joinParams.